core/lib/dal/src/blocks_web3_dal.rs

```diff
@@ -87,7 +88,10 @@ impl BlocksWeb3Dal<'_, '_> {
                         .unwrap_or(i64::from(LEGACY_BLOCK_GAS_LIMIT))
                         as u64)
                         .into(),
-                    // TODO: include logs
+                    logs_bloom: row
```

Comment:
Just to understand better: Is returning a dummy value here better than not returning a value at all, because it won't break Web3 clients?

Reply:
exactly